packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -152,6 +154,25 @@ export default class RestApi {
     })
 
     api.get('/', (req, res) => sendObjects(collections, req, res))
+
+    api
+      .get('/backups', (req, res) => {
+        sendObjects([{ id: 'logs' }], req, res)
+      })
+      .get(
+        '/backups/logs',
+        wrap(async (req, res) => {
+          const logs = await app.getBackupNgLogsSorted({})
```

Comment:
this will expose `restore`  `jobs` (= backups if my understanding is correct) , and `metdataRestore` in the same endpoint. I think we should have differents endpoint, since for the user backup logs  shouldn't contains the restores. 

Or implement filters ?